paimon-core/src/main/java/org/apache/paimon/catalog/Catalog.java

```diff
@@ -158,6 +159,19 @@ default boolean tableExists(Identifier identifier) {
     void createTable(Identifier identifier, Schema schema, boolean ignoreIfExists)
             throws TableAlreadyExistException, DatabaseNotExistException;
 
+    /**
+     * Retrieve the properties of a Hive table.
+     *
+     * <p>NOTE: Properties of system tables cannot be retrieved.
+     *
+     * @param identifier The identifier of the table whose properties are to be retrieved.
+     * @return A map containing the properties of the table.
+     * @throws TableNotExistException If no table exists with the given identifier.
+     * @throws DatabaseNotExistException If the database specified in the identifier does not exist.
+     */
+    Map<String, String> getTableParameters(Identifier identifier)
```

Comment:
This PUBLIC interface is only for testing. And it is only for Hive.

You should remove it and get Hive table from Hive api in testing.

Reply:
done.